modules/markup/renderer.go

```diff
@@ -179,7 +179,8 @@ func (nopCloser) Close() error { return nil }
 
 func renderIFrame(ctx *RenderContext, renderer Renderer, input io.Reader, output io.Writer) error {
 	_, err := io.WriteString(output, fmt.Sprintf(`<iframe src="%s%s/%s/render/%s/%s" name="ifd"
-onload="this.height=ifd.document.body.scrollHeight" width="100%%" scrolling="no" frameborder="0" style="overflow: hidden"></iframe>`,
+onload="this.height=ifd.document.body.scrollHeight" width="100%%" scrolling="no" frameborder="0" style="overflow: hidden"
+sandbox="allow-same-origin allow-scripts"></iframe>`,
```

Comment:
Doesn't `allow-scripts` defeat the whole point of trying to prevent XSS attacks?

Reply:
IIRC allow-scripts allows JS to be run inside the iframe, but doesn't allow access to the parent. (this is only an assumption though based on my reading of MDN, I'd be very happy to be corrected).

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#scripting :
> From the inside of a frame, a script can get a reference to its parent window with [window.parent](https://developer.mozilla.org/en-US/docs/Web/API/Window/parent).

The closest thing I found regarding your point:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-sandbox :
> allow-scripts: Lets the resource run scripts (but not create popup windows).

This sounds to me like the only thing blocked are pop-ups (`window.alert`, `window.confirm`, ...), but nothing else...